piff/input.py

```diff
@@ -996,8 +999,6 @@ def safe_to_image(wcs, ra, dec):
                 satur = float(header[satur])
                 logger.debug("Using saturation from header: %s",satur)
             extra_props['satur'] = np.array([satur]*len(cat), dtype=float)
-        else:
-            extra_props['satur'] = np.array([np.inf]*len(cat), dtype=float)
```

Comment:
I don't understand this block, not because I think it is wrong, but because I don't know this code base. I am pointing this out to say that I am not qualified to review this bit I think.